src/datastore/NedbDataStore.ts

```diff
@@ -483,39 +466,44 @@ export class DataStore {
         log.info("storeAdminRoom (id=%s, user_id=%s)", room.getId(), userId);
         room.set("admin_id", userId);
         await this.roomStore.upsertEntry({
-            id: DataStore.createAdminId(userId),
+            id: NeDBDataStore.createAdminId(userId),
+            matrix_id: room.getId(),
```

Comment:
Having `matrix_id: room.getId()` and `matrix: room` seems fragile? Is this needed?